src/panels/config/logs/error-log-card.ts

```diff
@@ -42,32 +44,64 @@ class ErrorLogCard extends LitElement {
 
   @property({ type: Boolean, attribute: true }) public show = false;
 
-  @state() private _isLogLoaded = false;
+  @query(".error-log") private _logElement?: HTMLElement;
+
+  @query("#scroll-marker") private _scrollMarkerElement?: HTMLElement;
+
+  @state() private _log?: string;
+
+  @state() private _scrolledToBottom?: boolean;
 
-  @state() private _logHTML?: TemplateResult[] | TemplateResult | string;
+  @state() private _scrolledToBottomObserver?: IntersectionObserver;
```

Comment:
Use `@lit-labs/observers/intersection-controller` for this.

https://github.com/lit/lit/tree/main/packages/labs/observers#readme